vm_setup_scripts/init-run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# This script is used to setup Driod VM template base on Rocky 8 Linux 8.6
+
+# install dependencies for test tools
+dnf update -y
+dnf install perl -y
+dnf --enablerepo=powertools install perl-List-MoreUtils -y
+dnf --enablerepo=powertools install perl-Readonly -y
+
```

Comment:
Where is test tool?

Reply:
Test tool will be installed during test as before.

Reply:
you need to update doc https://info.citrite.net/pages/viewpage.action?pageId=1604229299 to ensure consistent

Reply:
Sure, has updated the doc, Thanks.

Reply:
`, and user needs to install them to VM manually` should be removed.

Reply:
Has removed